tests/examples/test_scripts.py

```diff
@@ -94,6 +97,11 @@ def run_test(filepath):
             "translation.py",
             marks=pytest.mark.skipif(not _TEXT_AVAILABLE, reason="text libraries aren't installed")
         ),
+        pytest.param(
+            "finetuning",
+            "style_transfer.py",
+            marks=pytest.mark.skipif(not _IMAGE_STLYE_TRANSFER, reason="pystiche is not installed")
+        ),
```

Comment:
So this should run with every PR run? Each run will take ~2 hours.

Reply:
It is running with `FLASH_TESTING=1` which maps to `fast_dev_run=True` within the trainer.